packages/sdk/src/mcp/agents/api.ts

```diff
@@ -79,6 +79,7 @@ export const listAgents = createTool({
   name: "AGENTS_LIST",
   description: "List all agents",
   inputSchema: z.object({}),
+  outputSchema: z.array(AgentSchema),
```

Comment:
I think output schema can not be an array. Take a look at:

https://github.com/modelcontextprotocol/typescript-sdk/pull/528